portfolio/src/test/java/com/google/sps/CuratedPostsTest.java

```diff
@@ -0,0 +1,122 @@
+package com.google.sps.servlets;
+
+import static com.google.appengine.api.datastore.FetchOptions.Builder.withLimit;
+import static org.junit.Assert.assertEquals;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CuratedPostsTest {
+
+  private final LocalServiceTestHelper helper =
+    new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
+
+  @Before
+  public void setUp() {
+    helper.setUp();
+  }
+
+  @After
+  public void tearDown() {
+    helper.tearDown();
+  }
+
+  private void createDatastoreEntities() {
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+
+    Entity noTagEntity = new Entity("UserLogin");
+    noTagEntity.setProperty("id", "357911");
+    noTagEntity.setProperty("nickname", "noTags");
+    noTagEntity.setProperty("email", "noTags@testing.com");
```

Comment:
As much as possible, avoid bare strings for values that you use in multiple places. For any value that you compare against in your tests, you should have a constant for it. The fields that aren't compared are fine as bare strings. Using a constant creates a clear link between the setup for the test and how it is used in the actual testing.

You should also use the BlogConstants for the properties names. If they ever change, then a person only needs to change the one file without changing your test.